packages/venia-concept/src/util/makeUrl.js

```diff
@@ -1,8 +1,9 @@
 // If the root template supplies the backend URL at runtime, use it directly
-const {
-    imageOptimizingOrigin,
-    mediaBackend = 'https://backend.test/media/'
-} = document.querySelector('html').dataset;
+const htmlDataset = document.querySelector('html').dataset;
+const { imageOptimizingOrigin } = htmlDataset;
+// Protect against potential falsy values for `mediaBackend`.
+const mediaBackend = htmlDataset.mediaBackend || 'https://backend.test/media/';
```

Comment:
This fixes a bug identified where `mediaBackend` was an empty string which destructures as `""` instead of being falsy.

TBH I'm not sure why we have a default value here anyways -- seems like it just gets rewritten in the makeUrl code.

Reply:
Yeah, there shouldn't be a default here.

Reply:
Ok -- rather than remove the default, @zetlen and I discussed and it is out of scope. That being said we should probably rewrite `makeUrl` or at least fix this to not need a fallback (ie. console.error and THEN use a default.

Reply:
I just added a warning if it's falsy to at least indicate a problem.